utils/rowgroup/rowstorage.cpp

```diff
@@ -666,6 +666,156 @@ class RowGroupStorage
     }
   }
 
+  /** @brief Used to output aggregation results from memory and disk in the current generation in the form of
+   * RGData. Returns next RGData, loads from disk if necessary. Skips finalized rows as they would contain
+   * duplicate results, compacts actual rows into start of RGData and adapts number of rows transmitted in
+   * RGData.
+   * @returns A pointer to the next RGData or an empty pointer if there are no more RGDatas in this
+   * generation.
+   */
+  bool getNextOutputRGData(std::unique_ptr<RGData>& rgdata)
+  {
+    if (UNLIKELY(fRGDatas.empty()))
+    {
+      fMM->release();
+      return false;
+    }
+
+    while (!fRGDatas.empty())
+    {
+      uint64_t rgid = fRGDatas.size() - 1;
+      rgdata = std::move(fRGDatas[rgid]);
+      fRGDatas.pop_back();
+
+      // Calculate from first and last uint64_t entry in fFinalizedRows BitMap
+      // which contains information about rows in the RGData.
+      uint64_t fgid = rgid * fMaxRows / 64;
```

Comment:
magic number, I'd replace with constant or sizeof expression

Reply:
It was like this before but I will make it a const.